ethereum/jsonrpc/src/test/java/tech/pegasys/pantheon/ethereum/jsonrpc/internal/methods/DebugTraceTransactionTest.java

```diff
@@ -85,7 +85,8 @@ public void shouldTraceTheTransactionUsingTheTransactionTracer() {
             EnumSet.noneOf(ExceptionalHaltReason.class),
             Optional.empty(),
             Optional.empty(),
-            Optional.empty());
+            Optional.empty(),
+            "revert message");
```

Comment:
Can you add a test for the case where no transaction is found?

Reply:
Done